tree/ntuple/v7/inc/ROOT/RNTupleReader.hxx

```diff
@@ -300,6 +336,29 @@ public:
       return ROOT::RNTupleView<T>(std::move(field), range, objPtr);
    }
 
+   /// Provides access to an individual (sub)field from its on-disk ID, reading its values into `objPtr` as the type
+   /// provided by `typeName`.
+   ///
+   /// \sa GetView(std::string_view, std::shared_ptr<T>)
+   template <typename T>
+   ROOT::RNTupleView<T> GetView(ROOT::DescriptorId_t fieldId, std::shared_ptr<T> objPtr, std::string_view typeName)
+   {
+      static_assert(std::is_void_v<T>, "calling GetView with a type name string is only allowed for [T = void]");
```

Comment:
Actually, if we only allow `T = void`, does it make sense to have this function templated at all? It would deviate from other overloads, but I find it less surprising than the documentation telling me it's a template function and then being presented a `static_assert`...

Reply:
Yeah this is actually something I was asking myself as well when I was writing the docs. Hooking into your other comment, I don't think it is used in framework code so until it is I'm also fine removing it here.